### PR TITLE
Added test for bug in extension name computation

### PR DESCRIPTION
**Pull request by tivervac**

### CODE REVIEW

**Comment by tivervac** [comment]
Test for https://github.com/eclipse/xtext-xtend/issues/514

**Comment by cdietrich** [comment]
i wonder of you can provide a commit that uses the upstream from xtext-extras to actually test this.
the `adjustPipelines.sh` in umbrella might help to do the changes

**Comment by tivervac** [comment]
Will do on monday!

File: org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/quickfix/QuickfixTest.xtend [hunk 1]
```diff
@@ -123,6 +123,39 @@ class QuickfixTest extends AbstractXtendUITestCase {
 			''')
 	}
 
+	// see https://github.com/eclipse/xtext-xtend/issues/514
+	@Test
+	def void testBug514() {
+		create('test/Foo.xtend', '''
```
Comment (szarnekow) [inline org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/quickfix/QuickfixTest.xtend]: Could you please add a test with an assignment?

Comment (tivervac) [reply org.eclipse.xtend.ide.tests/longrunning/src/org/eclipse/xtend/ide/tests/quickfix/QuickfixTest.xtend]: Will do

**Comment by tivervac** [comment]
@cdietrich Is the last commit what you expected?

**Comment by cdietrich** [comment]
yes thats what i meant. once the extras build ran successfully we can see what happens on xtend

**Comment by cdietrich** [comment]
@tivervac can you please rebase so that we can see if tests are fine now

**Comment by cdietrich** [comment]
looks so http://services.typefox.io/open-source/jenkins/job/xtext-xtend/job/extension_name_computation/

**Comment by cdietrich** [comment]
please undo the upstream changes. then it can be merged i think

**Pull request merged**